### PR TITLE
[EventHubs] remove running loop in constructor

### PR DESCRIPTION
**Pull request by yunhaoling**
Issue #9269

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/eventhub/azure-eventhub/azure/eventhub/aio/_consumer_async.py [hunk 1]
```diff
@@ -74,15 +74,14 @@ def __init__(self, client: "EventHubConsumerClient", source: str, **kwargs) -> N
...
-        loop = kwargs.get("loop", None)
 
         self.running = False
         self.closed = False
 
         self._on_event_received = kwargs[
             "on_event_received"
         ]  # type: Callable[[EventData], Awaitable[None]]
-        self._loop = loop or get_running_loop()
+        self._loop = kwargs.get("loop", None)
```
Comment (YijunXieMS) [inline sdk/eventhub/azure-eventhub/azure/eventhub/aio/_consumer_async.py]: kwargs.get("loop") returns None if there is no "loop"

**Pull request merged**